src/main/scala/mesosphere/marathon/api/v2/PodsResource.scala

```diff
@@ -31,4 +50,207 @@ class PodsResource @Inject() (
   def capability(@Context req: HttpServletRequest): Response = authenticated(req) { _ =>
     ok()
   }
+
+  // TODO(jdef) force parameter? probably since this supports deployment; but what does it actually
```

Comment:
In /v2/apps you can create a deployment with multiple apps. `force=true` will kill this deployment and start a new one.